message.go

```diff
@@ -304,6 +304,11 @@ type MessageReference struct {
 	GuildID   string `json:"guild_id"`
 }
 
+// String returns link to message
+func (m *Message) String() string {
+	return "https://discordapp.com/channels/" + m.GuildID + "/" + m.ChannelID + "/" + m.ID
```

Comment:
If I were passing a `*Message` into something that accepts a `fmt.Stringer`, I think I would expect that it at least contains the content of the message instead of a URL.

Something like the following?

```suggestion
        return fmt.Sprintf("Message in %s (%s): %s", m.GuildID, m.ChannelID, m.Content)
```

Reply:
I think URL is better, ‘cause it can be used, for example, for logger bot

Reply:
even if you use it as a debug, what these numbers will give, so the URL will be more useful